portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,45 @@
+package com.google.sps.data;
+
+import java.util.Date;
+
+/** Class containing the data for comments. */
+public final class Comment {
+
+  private final long id;
+  private final String author;
+  private final String comment;
+  private final String timeStamp;
```

Comment:
would a java.util.Date type make sense for the timestamp?

Reply:
It would, but I just want a basic date in this format so that is why I have it as a string: MM/DD/YYYY